.NET/Microsoft.Recognizers.Definitions/German/DateTimeDefinitions.cs

```diff
@@ -1,4 +1,6 @@
-﻿//------------------------------------------------------------------------------
+﻿
```

Comment:
Nit: Be careful not to push this change. Not sure why it happens sometimes when VS regenerates the .cs file